update create parameter topic (#269)
DevExpress/dotnet-eud: eud-for-devexpress-reports/reporting-for-web/report-designer/use-report-parameters/create-a-report-parameter.md

### Create From the Properties Panel

> Select a report, navigate to the [Properties panel](../report-designer-tools/ui-panels/properties-panel.md) *Data* section, and click the ellipsis button in the *Parameters* editor.

Reviewer: ```suggestion
Select a report, go to the *Data* section of the [Properties panel](../report-designer-tools/ui-panels/properties-panel.md), and click the ellipsis button in the *Parameters* editor.
```

![Create a parameter from the Properties panel](../../images/property-grid-add-parameter.png)

> In the invoked **Edit Parameters** dialog, you can create a parameter, specify [parameter options](#parameter-options), add separators and groups, reorder, and delete parameters.

Reviewer: ```suggestion
In the **Edit Parameters** dialog, you can create parameters, configure [parameter options](#parameter-options), add separators and groups, reorder parameters, and delete them.
```
